sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -386,26 +386,50 @@ export class AssessmentControllerService {
       applicationId,
       options?.studentId,
     );
-    return requestAssessmentSummary.concat(appeals);
+    const applicationOfferingChangeRequests =
+      await this.getApplicationOfferingChangeRequestsByStatus(
+        applicationId,
+        [
+          ApplicationOfferingChangeRequestStatus.InProgressWithStudent,
+          ApplicationOfferingChangeRequestStatus.InProgressWithSABC,
+          ApplicationOfferingChangeRequestStatus.DeclinedByStudent,
+          ApplicationOfferingChangeRequestStatus.DeclinedBySABC,
+        ],
+        { studentId: options?.studentId },
+      );
+    return requestAssessmentSummary
+      .concat(appeals)
+      .concat(applicationOfferingChangeRequests)
+      .sort(
+        // sort by the latest submitted request first.
```

Comment:
First, I do not recall why this method is not used for students, if anyone remembers, please let me know 😉.
Second, the basic sort method can be shared between the Ministry/Institution and Student. Not only to centralize the code but to enforce on future maintenance that there is a connection between both.
It can be a helper method in the controller service.
```ts
sortAssessmentHistory(first: {submittedDate: Date}, second: {submittedDate: Date}): number {
  return secondRequest.submittedDate.getTime() - firstRequest.submittedDate.getTime();
}
```